helper/identity/templating.go

```diff
@@ -61,7 +68,7 @@ func PopulateString(p *PopulateStringInput) (bool, string, error) {
 		case 2:
 			subst = true
 			if !p.ValidityCheckOnly {
-				tmplStr, err := performTemplating(p.Namespace, strings.TrimSpace(splitPiece[0]), p.Entity, p.Groups)
+				tmplStr, err := performTemplating(p.Namespace, strings.TrimSpace(splitPiece[0]), p.Entity, p.Groups, p.AllowMissingSelectors)
```

Comment:
Might be cleaner to make a struct that can hold these values, or re-use `p` but put the string to template with in a variable before calling this?

Reply:
Decided to pass in the `p` and the string piece.